base/all-all/sbardef.lmp

```diff
@@ -874,6 +874,34 @@
                 {
                   "condition": 19,
                   "param": 6
+                },
+                {
+                  "condition": 20,
+                  "param": 0
+                }
+              ],
+              "children": null
+            }
+          },
+          {
+            "widget":
```

Comment:
> but this isn't going to scale well.

Why not, it's a very small tree compared to typical Doom levels for example. I doubt we'll have many of these conditions/copies. Yes, it can't effectively replace the functionality of WOOFHUD, but I think it's good enough.

Reply:
I think it's too much hard-coded to the default layout. For example, if I decide to move the level stats to the upper left, but leave the level time in the bottom left, will this work with that approach?